mart/configs/attack/adversary.yaml

```diff
@@ -2,6 +2,7 @@ defaults:
   - /callbacks@callbacks: [progress_bar]
 
 _target_: mart.attack.Adversary
+_convert_: all
```

Comment:
I feel like we should just convert everything in configuration by default. The use of `DictConfig` and `ListConfig` has tripped me up many times at this point.